content/articles/run-length-encoding/index.md

```diff
@@ -0,0 +1,150 @@
+### Introduction
+
+In this article, we will look at how Run Length Encoding works in helping us reduce our file sizes.
+Before delving into the topic, we will first get to know what is compression.
+
+**Compression** - This is a process where a file size is reduced using algorithms resulting in a file that uses fewer storage bits than the original file.
+For example, in a situation where one wants to send a picture to his/her friend, compression will be done at the source device and decompressed at the destination device. This enables the file to be sent faster and additionally, it reduces overhead traffic for transmitting the data.
+
+Another example of compression is in social media apps. It is applied in applications such as WhatsApp where one will notice the image he/she sent is of lower quality and consumes much lesser space. Other apps also give users options in data saving settings for whether choosing whether the images and videos loaded should be of low quality or high quality. 
+
+We also notice compression in video streaming sites where videos are loaded in low quality when there is low internet connectivity and vice versa. 
```

Comment:
There are more than 1 example. So, I would suggest you to add them as bulleted points under the definition